src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -1,5 +1,6 @@
 CREATE EXTENSION "uuid-ossp";
 
+-- We think there should be a field for para's tables to hold CM id's
```

Comment:
We do need a new table, say "case_manager", that contains the links for para_ids and student_ids. Something like 

// Not sure about postgres syntax, so here is the pseudo code:
Table case_manager (
  para_id - array of user_ids that are a foreign key into "user"
  student_id - array of student_ids that are a foreign key into "student"
  // ... other fields in the future
)

Reply:
Do you think this is a task for this PR, or can we make a new PR to handle this?

Reply:
Ideally, that table would be created in either this PR or in Hieu's PR to manage student roster, whichever happens to come first :). Those two PRs will add the para_id[] and student_id[] fields respectively in the table above. 

Another option is to create the table in a separate PR, and rebase this PR on that, but in general it is useful for a PR to be fully self-contained for a particular piece of functionality.

Without that table, this PR then becomes a way to create users in the database (and currently delete but see comment below) without associating them with a CM, which is orthogonal to where we want the CM dashboard to evolve.